From 3324dc771fa6c8f4d705d3ab23ea382be60d49bf Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Tue, 11 Nov 2025 09:33:48 +0100
Subject: [PATCH 1/2] update release date
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
RH-MergeRequest: 13: seabios updates for 9.8
RH-Jira: RHEL-131918 RHEL-131921
RH-Acked-by: Daniel P. Berrangé <berrange@redhat.com>
RH-Acked-by: Jon Maloy <jmaloy@redhat.com>
RH-Commit: [1/2] e90d4319ee2241c546479cdcf1ea7c6ca1e011eb (kraxel.rh/centos-src-seabios)

upstream release 1.16.3 was tagged on Nov 5th 2023,
so use that as release date on smbios table 0

Resolves: RHEL-131918

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 src/fw/biostables.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/fw/biostables.c b/src/fw/biostables.c
index 1c09a785..a81e99e7 100644
--- a/src/fw/biostables.c
+++ b/src/fw/biostables.c
@@ -483,7 +483,7 @@ smbios_new_type_0(void *start,
 }
 
 #define BIOS_NAME "SeaBIOS"
-#define BIOS_DATE "04/01/2014"
+#define BIOS_DATE "11/05/2023"
 
 /*
  * Build tables using qtables as input, adding additional type 0
-- 
2.51.1

